xyz

```diff
@@ -31,6 +31,11 @@ Options:
         'X.Y.Z' acts as a placeholder for the version number.
         'Version X.Y.Z' is assumed if this option is omitted.
 
+   --prerelease-label <label>
+        Specify the label to be used in the version number when publishing
+        a pre-release version (e.g. 'beta' is the label in '2.0.0-beta.0').
+        'rc' is assumed if this option is omitted.
+
```

Comment:
This addition shows that we have never fully supported the `pre{major,minor,patch,release}` increments.

Reply:
@Avaq, do you agree with `rc` as the default value?